docs/src/GettingStarted/Terminology.md

```diff
@@ -0,0 +1,40 @@
+# Terminology
+
+The `ClimateMachine` documentation uses terminology from several disciplines. Below are some definitions of some of these terms. Please let us know via a comment if there are others that you find confusing, and we will add them.
+
+## Software terms
+
+* Driver file: a julia script which (a) specifies which [`BalanceLaw`](#ClimateMachine.BalanceLaws.BalanceLaw) is being used, (b) specifies changes to the default version of that model, (c) specifies initial and boundary conditions, (d) specifies numerical details related to timestep, grid, and frequency of output, and (e) runs the integration. Examples can be found in tutorials and experiments.
```

Comment:
these types of links dont work. TBD
also would be nice to bold the terms being defined.

Reply:
I think you need [`BalanceLaw`](@ref ClimateMachine.BalanceLaws.BalanceLaw)

Reply:
this ended up working...
[`BalanceLaw`](../APIs/BalanceLaws/BalanceLaws.md#ClimateMachine.BalanceLaws.BalanceLaw)

@ref gave an error? not sure, that works in tutorials!

Reply:
> this ended up working...
> [`BalanceLaw`](../APIs/BalanceLaws/BalanceLaws.md#ClimateMachine.BalanceLaws.BalanceLaw)
> 
> @ref gave an error? not sure, that works in tutorials!